guides/mapping-and-blending/blending/README.md

```diff
@@ -1,4 +1,97 @@
-# Blending
+# Introduction
```

Comment:
The top headline should still be "Blending". Either place the introduction under the main heading or do something like this:

```
# Blending
## Introduction
...
## Blending token
```